docs/appendices/migration-guides/index.mdx

```diff
@@ -12,6 +12,43 @@ import ContactLink from "@site/src/components/ContactLink";
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
+## `2.18.0` -> `2.19.0`
+
+### Adding custom icons
+
+We have improved the way you can add custom icons to your theme. Before you had
+to override the
+[`theme/components/atoms/Icon/Icon.js`](https://gitlab.com/front-commerce/front-commerce/-/blob/main/src/web/theme/components/atoms/Icon/Icon.js)
+file to add your custom icons. Now you can simply extend or override the core
+icons with your custom icons by adding them to the
+[`theme/components/atoms/Icon/icons/app-icons.js`](https://gitlab.com/front-commerce/front-commerce/-/blob/main/src/web/theme/components/atoms/Icon/icons/app-icons.js)
+file.
+
+```js title="theme/components/atoms/Icon/icons/app-icons.js"
+import { FcHome, FcPhone } from "react-icons/fc";
+
+/** @type {Record<string, import("react-icons").IconType>} */
+const componentIcons = {
+  home: FcHome // this will override the `IoIosHome` icon from the core implementation
+  phone: FcPhone // this will add a new icon
+};
+
+/** @type {string[]} */
+const svgIcons = [];
+
+export default {
+  componentIcons,
+  svgIcons,
+};
+```
+
+:::important
+
+To apply this change you need to restore the original `Icon.js` files, you can
+do this by simply deleting the override from your custom theme.
```

Comment:
> **simple, simply**
>
> What might be simple for you might not be simple for others. Try eliminating this word from the sentence because usually the same meaning can be conveyed without it.

Source: https://developers.google.com/style/word-list#:~:text=simple%2C%20simply,conveyed%20without%20it.

```suggestion
do this by deleting the override from your custom theme.
```